aim/aim_integration.py

```diff
@@ -0,0 +1,59 @@
+"""
+Optuna example that optimizes a classifier configuration for the Iris dataset using
+scikit-learn and records hyperparameters and metrics using aim.
+
+In this example we optimize random forest classifier for the Iris dataset. All
+hyperparameters and metrics will be logged to aim via integration callback.
+
+You can run this example as follows:
+    $ python aim_integration.py
+
+Results and plots will be available in aim ui once script finishes:
+    $ aim ui
```

Comment:
A nit, but this command failed. According to README.md, it may be a typo for `aim up`.
```suggestion
    $ aim up
```

Reply:
Nice catch! Thanks!